app/src/main/java/com/example/android/weatherapp/models/ThreeHourlyWeatherModel.kt

```diff
@@ -1,6 +1,6 @@
 package com.example.android.weatherapp.models
 
-data class ThreeHourlyWeatherModel (
+data class ThreeHourlyWeatherModel(
```

Comment:
Add a space before parenthesis.

Reply:
https://kotlinlang.org/docs/reference/data-classes.html
here there is no space before parenthesis. Should i still add the space?